config/Config-kernel.in

```diff
@@ -1175,6 +1181,13 @@ if KERNEL_IPV6
 	config KERNEL_LWTUNNEL_BPF
 		def_bool n
 
+	config KERNEL_MPTCP_IPV6
+		bool "Enable IPv6 MultiPath TCP support"
+		default n
```

Comment:
```suggestion
		default y
```
I assume that most users that have selected `KERNEL_MPTCP` also want to have IPv6 support. 
Please make this active by default.